packages/react-native-editor/metro.config.js

```diff
@@ -0,0 +1,24 @@
+/** @format */
+const path = require( 'path' );
+const enm = require( './extra-node-modules.config.js' );
+
+module.exports = {
+	watchFolders: [
+		path.resolve( __dirname, '../..' )
+	],
+	resolver: {
+		sourceExts: [ 'js', 'json', 'scss', 'sass', 'ts', 'tsx' ],
+		extraNodeModules: enm,
+	},
+	transformer: {
+		babelTransformerPath: require.resolve( './sass-transformer.js' ),
+		getTransformOptions: async () => ( {
+			transform: {
+				experimentalImportSupport: false,
+				inlineRequires: false,
+			},
+		} ),
+	},
+};
+
+console.log( 'metro config ', module.exports );
```

Comment:
Do you want to leave it here or is it temporary?

Reply:
Good point, this should be removed